apis/metal3.io/v1alpha1/hostfirmwaresettings_types.go

```diff
@@ -53,6 +53,12 @@ type HostFirmwareSettingsStatus struct {
 
 	// Settings are the actual firmware settings stored as name/value pairs
 	Settings SettingsMap `json:"settings" required:"true"`
+
+	// Stores the time that the settings were last updated
+	LastUpdated *metav1.Time `json:"lastUpdated,omitempty"`
+
+	// Track whether settings stored in the spec are valid based on the schema
+	IsValid bool `json:"isValid"`
```

Comment:
As a further comment, and also to keep aligned the design adopted for sub-resources, this part looks like a good candidate to be implemented with `Conditions`(see the current work developed by @zaneb  on [PR 936](https://github.com/metal3-io/baremetal-operator/pull/936/files#diff-875ad84c1b2d03186f5fca344650cb756732a75844100709818981560a76e345R81)).
More recommendations about `Conditions` usage can be found [here](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties)

Reply:
Good idea and thanks for the links. That seems to be a standard way of providing info regarding the resource. I've added 3 conditions for changeRequested, Valid, and Error which can be used by the bmh to get the status of the resource.